spec-dtslint/operators/windowCount-spec.ts

```diff
@@ -0,0 +1,15 @@
+import { of } from 'rxjs';
+import { windowCount } from 'rxjs/operators';
+
+it('should infer correctly', () => {
+  of('test').pipe(windowCount(1)); // $ExpectType Observable<Observable<string>>
```

Comment:
Maybe add another call here that specifies an argument for `startWindowEvery`? For completeness?

Reply:
Certainly. Will do.